wandb/sdk/wandb_settings.py

```diff
@@ -1424,7 +1424,7 @@ def _infer_run_settings_from_environment(
         program = self.program or _get_program()
         if program is not None:
             program_relpath = self.program_relpath or _get_program_relpath_from_gitrepo(
-                program, _logger=_logger
+                program, git_disabled=self.disable_git, _logger=_logger
```

Comment:
@dmitryduev could this dependency cause an issue since we are using the `disable_git` value, although I see `program` and `program_relpath` as well, so it might be actually fine

Reply:
Should be fine I guess.